tests/network/vmi_multus.go

```diff
@@ -675,8 +675,6 @@ var _ = Describe("[Serial]SRIOV", func() {
 		virtClient, err = kubecli.GetKubevirtClient()
 		util.PanicOnError(err)
 
-		tests.SkipIfNonRoot(virtClient, "SRIOV")
```

Comment:
do we have a test lane that is validating this works in the non-root case?

Reply:
Yes, we have following:
pull-kubevirt-e2e-kind-1.19-sriov-nonroot
pull-kubevirt-e2e-kind-1.22-sriov-nonroot
pull-kubevirt-e2e-kind-1.19-vgpu-nonroot